app/controllers/facilities_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FacilitiesController < ApplicationController
+  def new
+    @facilities = Facility.new
+  end
+
+  def create
+    @facility = Facility.new facility_params
+    if @facility.save
+      flash[:notice] = "New facility created!"
+      redirect_back(fallback_location: new_facility_path)
```

Comment:
we keep it like this for a while, because we have not create a index root view yet.